.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 package
 .DS_Store
 .slack/apps.dev.json
+deno.lock
```

Comment:
I'm not sure we want to ignore the lockfile. Probably need a samples-wide discussion on what we should do here

Reply:
The lockfile was introduced in a recent version of deno. But we should talk about if samples should include them or not.